cli/src/command/account.rs

```diff
@@ -644,8 +664,20 @@ pub async fn node_info_command(account: &Account) -> Result<(), Error> {
 }
 
 /// `output` command
-pub async fn output_command(account: &Account, output_id: String) -> Result<(), Error> {
-    let output = account.get_output(&OutputId::from_str(&output_id)?).await;
+pub async fn output_command(account: &Account, selector: OutputSelector) -> Result<(), Error> {
+    let output = match selector {
+        OutputSelector::Id(id) => account.get_output(&id).await,
+        OutputSelector::Index(index) => {
+            let mut outputs = account.outputs(None).await?;
+            outputs.sort_by(|a, b| a.output_id.cmp(&b.output_id));
+            outputs.sort_by(|a, b| {
+                b.metadata
+                    .milestone_timestamp_booked()
+                    .cmp(&a.metadata.milestone_timestamp_booked())
+            });
```

Comment:
Sorting by ID is useless if you end up sorting by timestamp right after

Reply:
That's still no correct. You simply can't do two sort one after the other, the second will always cancel the first one. You need to sort by pair (timestamp, ID), a single sort

Reply:
Yes, I'm sure

Reply:
<img width="854" alt="image" src="https://github.com/iotaledger/iota-sdk/assets/3305068/32d60d1e-46ae-4fb2-90ca-c94cf5773eb6">
Example of it failing if you don't believe me

Reply:
Look at the date of the last entry, it's clearly wrong

Reply:
yes, I overlooked it at first.

Reply:
they should have been equivalent, but I am not fit today.